src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -217,31 +217,19 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
   /**
    * Creates a set of JES inputs for a backend call.
+   * Note that duplicates input files (same gcs path) will be (re-)localized every time they are referenced.
```

Comment:
It means that if you have a call with say
```
File a
File b
```
Where a and b both point to the same gcs path, there will be one `JesInput` for each, which makes me to believe that JES will localize them both, and the second one will overwrite the first one, which is fine but unnecessary.